gr-qtgui/lib/QFosphorColorMapper.cc

```diff
@@ -76,29 +83,24 @@ int QFosphorColorMapper::loadFromFile(QFile& file)
             addPalette(name, pixmap);
 
             /* Release name & prepare for next */
-            free(name);
-            name = NULL;
+            name.clear();
 
             kp = -1;
         } else {
-            float f[4];
+            std::array<float, 4> f;
 
             /* Read data point */
-            sscanf(line.constData(), "%f %f %f %f", f, f + 1, f + 2, f + 3);
+            ss.str() = line.toStdString();
```

Comment:
There was one more case of this needing to be ss.str(line.toStdString()); down here, sorry didn't call it out in the above comment. Though the color seems to work without this change, but should probably still make it

Reply:
I should have fixed that by myself, you shouldn't have to call out individual instances of systematic problems … sorry.